### PR TITLE
Remove group api as logic was moved to client

### LINKED ISSUE #48: Remove group API?
* UI handles group logic

### PR DESCRIPTION
**Pull request by georg-schwarz**
Closes #48.

@MBuchalik do you see any problems with this change?

### CODE REVIEW

File: client-generator/openapitools.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
       "ts-rxjs": {
         "generatorName": "typescript-rxjs",
         "output": "../client-generated/",
-        "inputSpec": "http://localhost:3000/api-json",
+        "inputSpec": "http://localhost:3001/api-json",
```
Comment (MBuchalik) [inline client-generator/openapitools.json:11]: Why this change? I believe the server is listening on port 3000 (at least in the local dev setup). The dev server for the frontend is on port 3001.

**Review by MBuchalik — COMMENTED** [review]
Except for the one question regarding the port, I don't see any issues 👍

File: client-generator/openapitools.json (hunk 1 above)
Comment (georg-schwarz) [reply client-generator/openapitools.json]: Good catch, adapted to some local changes. Reverted in [0a8c889](https://github.com/osrgroup/msadoc/pull/83/commits/0a8c8899ae7e8d41190b6e5eac5c7b9495fd2dcf)
